hvsock.go

```diff
@@ -136,21 +236,54 @@ func (l *HvsockListener) Accept() (_ net.Conn, err error) {
 	}
 	defer l.sock.wg.Done()
 
-	// AcceptEx, per documentation, requires an extra 16 bytes per address.
+	// AcceptEx, per documentation, requires an extra 16 bytes per address:
+	// https://docs.microsoft.com/en-us/windows/win32/api/mswsock/nf-mswsock-acceptex
 	const addrlen = uint32(16 + unsafe.Sizeof(rawHvsockAddr{}))
 	var addrbuf [addrlen * 2]byte
 
 	var bytes uint32
-	err = syscall.AcceptEx(l.sock.handle, sock.handle, &addrbuf[0], 0, addrlen, addrlen, &bytes, &c.o)
-	_, err = l.sock.asyncIo(c, nil, bytes, err)
+	err = syscall.AcceptEx(l.sock.handle,
+		sock.handle,
+		&addrbuf[0],
+		0, // rxdatalen
+		addrlen,
+		addrlen,
+		&bytes,
+		&c.o)
+	_, err = l.sock.asyncIo(c,
+		nil, // deadlineHandler
+		bytes,
+		err)
```

Comment:
I really don't know what the best practice for multi line function calls, but this is hard to read for me (it's likely a mix of two multiline functions with no newline in between also haha)